client/blocks/reader-feed-header/index.jsx

```diff
@@ -13,7 +13,7 @@ import config from 'config';
  */
 import { Card } from '@automattic/components';
 import ReaderFollowButton from 'reader/follow-button';
-import { isAuthorNameBlocked } from 'reader/lib/author-name-blocklist';
```

Comment:
Is blocklist troublesome here?

Reply:
I suppose not, it's probably one of the cases where `blocklist` is pretty clear actually. I'll revert this one.